packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -381,7 +381,13 @@ class FakeTicker implements Ticker {
 }
 
 class TestClipPaintingContext extends PaintingContext {
-  TestClipPaintingContext() : super(ContainerLayer(), Rect.zero);
+  factory TestClipPaintingContext() {
+     return TestClipPaintingContext._(ContainerLayer());
+  }
+
+  TestClipPaintingContext._(this._layer) : super(_layer, Rect.zero);
+
+  ContainerLayer _layer = ContainerLayer();
```

Comment:
Do you need this default value? Looks like in all code paths it's always overridden? Also, should this field be final?

Reply:
I need it just to dispose. It is created, so it needs to be disposed.